File: src/main/java/com/safjnest/model/BotSettings/BotSettings.java
package com.safjnest.model.BotSettings;

import java.awt.Color;
import java.util.List;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.safjnest.model.util.ColorDeserializer;

import lombok.Data;

@Data
public class BotSettings {
    private String name;
    private String discordToken;
    @JsonDeserialize(using = ColorDeserializer.class)
    private Color embedColor;
    private String prefix;
    private String activity;
    private String ownerId;
    private List<String> coOwnersIds;
    private String helpWord;
    private Integer maxPrime;
    private Integer maxFreePlaylists;
    private Integer maxFreePlaylistSize;
    private Integer maxPremiumPlaylists;
    private Integer maxPremiumPlaylistSize;
    private String info;
}

File: src/main/kotlin/com/safjnest/model/BotSettings/BotSettings.kt
package com.safjnest.model.BotSettings

import java.awt.Color

data class BotSettings(
    val name: String = "",
    val discordToken: String = "",
    val embedColor: Color = Color.BLUE,
    val prefix: String = "!",
    val activity: String = "",
    val ownerId: String = "",
    val coOwnersIds: List<String> = listOf(),
    val helpWord: String = "help",
    val maxPrime: Int = 10,
    val maxFreePlaylists: Int = 5,
    val maxFreePlaylistSize: Int = 100,
    val maxPremiumPlaylists: Int = 20,
    val maxPremiumPlaylistSize: Int = 500,
    val info: String = ""
)